### ISSUE TITLE
panic: runtime error: integer divide by zero

### ISSUE THREAD

**Issue by timhughes** [Percona-Lab/mysql_random_data_load#33]
Don't know what specifically caused this 

```
[root@ip-10-0-0-170 ~]# mysql_random_data_load --version  xx xx 2
Version   : v0.1.10
Commit    : 77ba5b493094d79318d941ea45f43268115c7be1
Branch    : master
Build     : 2018-10-10T16:39:41-0300
Go version: go1.11.1
```

```
panic: runtime error: integer divide by zero

goroutine 1 [running]:
main.main()
	/home/karl/go/src/github.com/Percona-Lab/mysql_random_data_load/main.go:187 +0x1536
```

**Comment by percona-csalguero** [comment]
`--bulk-size` param is 0.
It is fixed in v0.1.11 https://github.com/Percona-Lab/mysql_random_data_load/releases/tag/v0.1.11

**Comment by timhughes** [comment]
awesome , thanks
